src/main/kotlin/org/jitsi/nlj/RtpReceiverImpl.kt

```diff
@@ -94,6 +95,7 @@ class RtpReceiverImpl @JvmOverloads constructor(
     private val inputTreeRoot: Node
     private val incomingPacketQueue =
             PacketInfoQueue("rtp-receiver-incoming-packet-queue", executor, this::handleIncomingPacket, Config.queueSize())
+    private val incomingPacketQueueStats = QueueStatistics(incomingPacketQueue).also { incomingPacketQueue.setObserver(it) }
```

Comment:
Should `QueueStatistics` do this in its constructor? Would we ever want to construct `QueueStatistics` for a specific queue, but not set it as an observer?